login/core/validations/DbValidation.js

```diff
@@ -0,0 +1,22 @@
+const Sequelize = require('sequelize');
+
+const SequelizeOperators = Sequelize.Op;
+const UserModel = require('../../db/models/UserModel.js');
+const createUserAccount = require('../../db/CreateUserAccount.js');
+
+function CopyCheck(registrationRequest, response) {
```

Comment:
I'm not sure this function is supposed to make it into production with this name, the name of the file itself doesn't tell me much and is kinda confusing.